example.js

```diff
@@ -1,34 +1,14 @@
 const {
-  toHindi,
-  toShahmukhi,
   toAscii,
   toUnicode,
-  firstLetters,
   toEnglish,
-  stripAccents,
-  stripVishraams,
-  stripEndings,
-  isGurmukhi,
-  toSyllabicSymbols,
-  countSyllables,
+  stripEndings
 } = require( 'gurmukhi-utils' )
 
-// Encoding conversions
-console.log(toUnicode( 'Koj' ))
-console.log(toAscii('ਖੋਜ'))
+const unicodeGurmukhi = 'ਸੋ ਘਰੁ ਰਾਖੁ; ਵਡਾਈ ਤੋਇ ॥੧॥ ਰਹਾਉ ॥'
+const asciiGurmukhi = 'so Gru rwKu; vfweI qoie ]1] rhwau ]'
 
-// Gurmukhi line utilities
-console.log(firstLetters( 'ਹਰਿ ਹਰਿ ਹਰਿ ਗੁਨੀ' ))
-console.log(stripAccents('ਜ਼ਫ਼ੈਸ਼ਸ'))
```

Comment:
Why remove the exhaustive list of utilities?

`examples.js` specifies what default opens when you click "try in runkit" on the npm page, so the idea would be that they can just click play to see all the outputs and not have to enter them themselves

Reply:
Yes, that's exactly why I changed it. There's no test if you put everything there for them. Instead I've offered two strings which can be used with any of our functions. And there is a very clear to use API explaining what they can do in the RunKit.

It is possible to put every function in there, but as the list keeps growing, we may want to consider stopping that practice for two reasons:

1) It becomes quite overwhelming to see all that, and to even "run" it would take time once you've added that many functions

2) It becomes a burden on new contributors who may not be aware that they have to add every new function to example.js

In the end I consider the RunKit as a way to open up a playground that has gurmukhi utils ready to use. I don't think it's there to show what actually outputs for every single function in our API. As that's what our Docs already do? You don't have to hit a "run" button to see what will be outputted, because our docs already tell you.

Can change, of course, but just offering my alternative viewpoint